### ISSUE TITLE
Unusable GUI with Windows 10 and a fresh Binja 

### ISSUE THREAD

**Issue by marpie** [Vector35/binaryninja-api#1529]
Using a freshly installed Windows 10 (Pro) with a new BinaryNinja installation (stable or dev - shows the same issue).

The menu-bar is missing and the tab- and buttons seem to be *scaled*.

![image](https://user-images.githubusercontent.com/703785/74743820-60596c80-5261-11ea-9f91-bde9c3a2d0d1.png)

Binja Build: Version 1.3.2039-dev (Build ID b5f206e2)
Windows: 10.0.18363.657

**UPDATE**

Setting `updates.activeContent` (GUI: *Preferences* / *Updates* / **disable** *Active Content*) fixes the problem:

![image](https://user-images.githubusercontent.com/703785/74751083-0bbbee80-526d-11ea-81bc-71100e8cb08a.png)

**Comment by marpie** [comment]
It seems to work with an old version (Version 1.2.1921 (Build ID 4ca675f1))

![image](https://user-images.githubusercontent.com/703785/74745368-158d2400-5264-11ea-8d23-eab0485a3871.png)

**Comment by psifertex** [comment]
Thanks for testing -- likely a QT regression. You said to repro it was a brand-new win10 vm?

**Comment by marpie** [comment]
Yeah, I used the MS ISO yesterday and applied all updates, then installed Binja.

**Comment by psifertex** [comment]
Follow on to add that through some additional testing with marpie it appears that any of the QtWebEngine enabled builds are triggering this behavior. Trying to reproduce it now with a fresh build of our own.

**Comment by psifertex** [comment]
Thanks to @marpie's help we've narrowed this down to a bug with certain graphics cards. You can see more information here: https://stackoverflow.com/questions/45314416/pyqt-webengineview-interferes-with-mainmenu and https://jira.atlassian.com/browse/HCPUB-3177

TL;DR If you `set QT_OPENGL=angle`, it should resolve the issue. Going to go ahead and close this issue as it's not something we're going to investigate more but we will add to the windows troubleshooting documentation.

**Comment by marpie** [comment]
Just as a quick heads-up, the current Intel Drivers (27.20.100.8280 - 2020-05-19) seem to fix the issue completely. No workaround is needed anymore.
